src/WebJobs.Script/Binding/TableBinding.cs

```diff
@@ -184,11 +198,31 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes()
                 }
                 else
                 {
-                    // binding to entire table (query multiple table entities)
-                    RuntimeBindingContext runtimeContext = new RuntimeBindingContext(new TableAttribute(TableName, boundPartitionKey, boundRowKey), additionalAttributes);
+                    // binding to multiple table entities
+                    RuntimeBindingContext runtimeContext = new RuntimeBindingContext(new TableAttribute(TableName), additionalAttributes);
```

Comment:
I've updated the attribute generation for the binding to correctly handle pk/rk.